docs/content/routing/services/index.md

```diff
@@ -321,7 +321,38 @@ Below are the available options for the health check mechanism:
                 My-Custom-Header: foo
                 My-Header: bar
     ```
+    
+#### Response Forwarding
+
+Configure response forwarding to define how forwards serves back the response of requests to remote server.
```

Comment:
This section is about configuring how Traefik forwards the response from the backend server to the client.